go/interpreter/lfvm/super_instructions_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package lfvm
+
+import (
+	"testing"
+
+	"github.com/holiman/uint256"
+)
+
+func TestSI_opDup2_Lt(t *testing.T) {
+
+	tests := map[string]struct {
+		a, b   uint256.Int
+		result uint256.Int
+	}{
+		"a<b": {
+			a:      *uint256.NewInt(1),
```

Comment:
nit: use a variable for `*uint256.NewInt(1)` and zero, making it less verbose.

Reply:
I am going to merge this pr now because it is green. Using newInt is not that bad anyway.